.gitlab-ci.yml

```diff
@@ -122,36 +122,82 @@ publish:build:master:
     - docker
   dependencies:
     - build:raspbian_latest
+  script:
+    - echo "publishing image to Docker Hub"
+    - echo -n $DOCKER_HUB_PASSWORD | docker login -u $DOCKER_HUB_USERNAME --password-stdin
+    - docker load -i raspbianImage.tar
+    - docker push $DOCKER_REPOSITORY:raspbian_latest
+
+publish:acceptance-testing:
+  stage: publish
+  services:
+    - docker:dind
+  only:
+    - master
+  tags:
+    - docker
```

Comment:
would be even nicer in a publishing template I think - since it now exists 4 times 😉

Reply:
I consider it, but I thought it was little to save. But once you point it out... I cannot ignore :wink: